frontend/src/common/HelpPopover.spec.tsx

```diff
@@ -0,0 +1,36 @@
+import { render, screen, waitFor } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { describe, expect } from 'vitest'
+
+import { doNothing } from '../../utils/testUtils'
```

Comment:
This will be picked up if you merge in the latest version of `main`, but this won't be needed once you've removed the `onClose` props.

The latest version of `main` includes a lot of eslint changes that'll help to make these unused imports easier to find :)